src/issue.ts

```diff
@@ -74,17 +74,17 @@ async function getRepoIssues(repo: Repo, flags?: Flags): Promise<IssueResult> {
       repo: repo.repo,
       types: getTypes(r),
       api,
-      team: getTeam(r.Repo, api),
+      team: getTeam(r.repo, api),
       isOutOfSLO: isOutOfSLO(r),
       isTriaged: isTriaged(r),
-      pri: r.PriorityUnknown ? undefined : r.Priority,
+      pri: r.priorityUnknown ? undefined : getPriority(r.priority),
```

Comment:
Given this change and the method below, can you explain exactly what changed with the API?

Reply:
Sure! basically we are using a [proto enum field](https://github.com/GoogleCloudPlatform/devrel-services/pull/24/files#diff-c0752684c0c2862f8d029c8e99e9e8ccR63-R70) for the Prioroty now which the gRPC => HTTP translation serializes as a string and not as a number, necessitating the funciton

Reply:
Rad.  Just please make sure something like this ends up in a comment in that function :)